### ISSUE TITLE
Change spacing with double bond ?

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/openchemlib-js#242]
I have the issue printing small labels with large molecules:

![Image](https://github.com/user-attachments/assets/14047dfe-6e8c-46ec-9d65-92b8ed46fc1f)

The double bonds are not visible anymore.

I would like to be able to change the option `cFactorBondSpacing` but it seems to me to be private without any way to change it.

https://github.com/cheminfo/openchemlib/blob/17d264c81ccea9176ce576daedf959294eb93ff4/src/main/java/com/actelion/research/chem/AbstractDepictor.java#L148-L149

@targos Do you see a way we could customise this parameter that it not too hacky ?

**Comment by targos** [comment]
Not really. I would ask Thomas for suggestions.

**Comment by thsa** [comment]
To me it seems that we have a resolution issue here. The Depictor constants are optimized for publishing quality drawings, not to still recognize something when the resolution is reduced to the bare minimum. If you consider this really important, we can, of course, make the constant accessible...

**Comment by lpatiny** [comment]
Yes, it is clearly a resolution problem. We are printing on small labels using thermal transfer (Zebra printers, 300 dpi, resin ribbon), and we need to increase the bond width to keep the bonds readable and maintain high contrast. Giving the possibility to change the double bond distance would help in our case.

This is how to did the 'hack' to change the bondWidth:

https://github.com/cheminfo/openchemlib-js/blob/e624e76552e59515d73548fe2eb217608835693d/src/com/actelion/research/gwt/minimal/JSMolecule.java#L148-L150

**Comment by thsa** [comment]
would it be helpful to change the bondSpacing on a class level rather than for an Depictor instance?

As: public static void setBondSpacing(float bs) ?

**Comment by lpatiny** [comment]
A public static method on the Depictor instance seems perfect to me.

It is here that we would use it.

https://github.com/cheminfo/openchemlib-js/blob/e624e76552e59515d73548fe2eb217608835693d/src/com/actelion/research/gwt/minimal/JSMolecule.java#L171-L177

**Comment by thsa** [comment]
sorry for the delay. It is now done...
